helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/templates/clusterRole.yaml

```diff
@@ -1,19 +1,25 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: k8s-metadata-query-logging
+  name: {{ template "splunk-kubernetes-logging.fullname" . }}
```

Comment:
I tried this as well. but it fails with something like already existing cluster role name. that is why this name metadata is hardcoded.

Reply:
I ill do a re run and see